src/strategies/strategies/evolabs-dao/index.ts

```diff
@@ -107,22 +110,28 @@ export async function strategy(
       }
     });
   } else {
-    // Use Snapshot delegation system
-    const delegationSpace = options.delegationSpace || space;
-    const snapshotDelegations = await getDelegations(
-      delegationSpace,
-      network,
-      addresses,
-      snapshot as any
-    );
-
-    // Convert Snapshot delegations to our format
-    Object.entries(snapshotDelegations).forEach(([delegate, delegators]) => {
-      delegations[delegate] = delegators as string[];
-      (delegators as string[]).forEach(delegator => {
-        delegationMappings[delegator] = delegate;
+    // Use Snapshot delegation system with error handling
+    try {
+      const delegationSpace = options.delegationSpace || space;
+      const snapshotDelegations = await getDelegations(
+        delegationSpace,
+        network,
+        addresses,
+        snapshot as any
+      );
+
+      // Convert Snapshot delegations to our format
+      Object.entries(snapshotDelegations).forEach(([delegate, delegators]) => {
+        delegations[delegate] = delegators as string[];
+        (delegators as string[]).forEach(delegator => {
+          delegationMappings[delegator] = delegate;
+        });
       });
-    });
+    } catch (error) {
+      // If delegation subgraph is not available for this network, continue without delegation
+      console.warn(`Delegation not available for network ${network}: ${error}`);
+      // delegations and delegationMappings remain empty, so everyone votes for themselves
+    }
```

Comment:
This will return inconsistent voting power even if there is a network error, better throw if the delegation subgraph is not available with a `if`